src/components/VirtualList/_VirtualList.scss

```diff
@@ -0,0 +1,70 @@
+.pf-c-scrollablegrid .pf-c-table {
```

Comment:
Out of curiosity, why is the file prefixed with "_"? I see some file with the same, but most of them without.

Reply:
I saw it in a guide about good practices and I think that I am the only one who use styles by component, in fact the openshift team use _name too

Reply:
In that case I imported this file in the index card under styles folder